composer.json

```diff
@@ -94,6 +94,9 @@
             "bobthecow/mustache-bundle": {
                 "Remove superfluous argument": "https://github.com/bobthecow/BobthecowMustacheBundle/pull/9.patch"
             },
+            "cocur/slugify": {
```

Comment:
I guess this is urgent and no release will be out to include it soon enough

Reply:
Not urgent, but has been merged upstream so should be safe to include now.

Reply:
This is JSON so no comments are allowed but saying somewhere "remove when something newer than $current_version is available" could be useful to avoid having to cross-check versions the next time

Reply:
It will start to fail when it's updated.